WalletWasabi/WabiSabi/Backend/Rounds/RoundParameters.cs

```diff
@@ -19,15 +19,19 @@ public class RoundParameters
 			Random = random;
 			FeeRate = feeRate;
 
-			MaxInputCountByAlice = wabiSabiConfig.MaxInputCountByAlice;
+			MaxInputCountByRound = wabiSabiConfig.MaxInputCountByRound;
+			MinInputCountByRound = wabiSabiConfig.MinInputCountByRound;
 			MinRegistrableAmount = wabiSabiConfig.MinRegistrableAmount;
 			MaxRegistrableAmount = wabiSabiConfig.MaxRegistrableAmount;
+			MaxInputCountByAlice = wabiSabiConfig.MaxInputCountByAlice;
```

Comment:
Should not be relevant until #5656 is reversed (if at all)

Reply:
I removed all the related concepts.